### PR TITLE
[IA-4374] update wb libs versions

### PR DESCRIPTION
**Pull request by n8landolt**
__Jira ticket__: https://broadworkbench.atlassian.net/browse/IA-4374

### What

- Update workbench libs hash and also update an RC version

### Why

-

### What to test

- 

### Who tested and where

- [ ] This change is covered by automated tests 
  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [ ] I validated this change 
- [ ] Primary reviewer validated this change 
- [ ] I validated this change in the dev environment

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -131,7 +131,7 @@ object Dependencies {
   val sealerate: ModuleID =       "ca.mrvisser"         %% "sealerate"            % "0.0.6"
   val googleCloudNio: ModuleID =  "com.google.cloud"    % "google-cloud-nio"      % "0.126.15" % Test // brought in for FakeStorageInterpreter
 
-  val circeYaml =         "io.circe"          %% "circe-yaml"           % "0.14.2"
+  val circeYaml =         "io.circe"          %% "circe-yaml"           % "0.15.0-RC1"
```
Comment (n8landolt) [inline project/Dependencies.scala:135]: @Qi77Qi Is this fine to use an RC?
It fixes a vulnerability from a dependency of snakeyaml
Here's the RC being used:
https://mvnrepository.com/artifact/io.circe/circe-yaml_2.13/0.15.0-RC1

Here's the version that imports vulnerable versions.
https://mvnrepository.com/artifact/io.circe/circe-yaml_2.13/0.14.2

Comment (Qi77Qi) [reply project/Dependencies.scala:135]: Probably fine if automation tests are all passing

**Review by Qi77Qi — APPROVED** [review]
as long as tests pass 👍

**Review by mlilynolting — APPROVED** [review]

File: project/Dependencies.scala (hunk 1 above)
Comment (mlilynolting) [inline project/Dependencies.scala:135]: A comment here to explain the reason we're on an RC branch would be good.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -17,16 +17,16 @@ object Dependencies {
   val munitCatsEffectV = "1.0.7"
   val pact4sV = "0.9.0"
 
-  private val workbenchLibsHash = "01a11c3"
-  val serviceTestV = s"2.1-$workbenchLibsHash"
-  val workbenchModelV = s"0.17-$workbenchLibsHash"
+  private val workbenchLibsHash = "d764a9b"
+  val serviceTestV = s"3.1-$workbenchLibsHash"
```
Comment (mlilynolting) [inline project/Dependencies.scala:21]: This is a major version bump - is there a reason Leo's version of serviceTest was sitting at 2.1?

Comment (mlilynolting) [reply project/Dependencies.scala:21]: Resolved

File: project/Dependencies.scala (hunk 1 above)
Comment (mlilynolting) [reply project/Dependencies.scala:135]: done

**Review by mlilynolting — APPROVED** [review]

**Comment by mlilynolting** [comment]
jenkins retest

**Comment by mlilynolting** [comment]
jenkins retest

**Comment by mlilynolting** [comment]
Rerunning unit tests to see if that failure was ephemeral: https://github.com/DataBiosphere/leonardo/actions/runs/5533447116?pr=3477

**Comment by mlilynolting** [comment]
jenkins retest

**Comment by mlilynolting** [comment]
https://stackoverflow.com/questions/70565409/selenium-chrome-driver-error-java-lang-abstractmethoderror-receiver-class-org-o seems like a description of the same error message, though I'm not sure if the selected answer applies to us.

**Comment by mlilynolting** [comment]
jenkins multi-test

**Comment by mlilynolting** [comment]
jenkins retest

**Comment by mlilynolting** [comment]
jenkins retest

**Comment by mlilynolting** [comment]
jenkins retest

**Comment by mlilynolting** [comment]
jenkins retest

**Comment by mlilynolting** [comment]
jenkins retest

**Comment by mlilynolting** [comment]
jenkins retest

**Comment by mlilynolting** [comment]
Last run failed only on RuntimeDataprocSpec, Google permission propagation! Fingers crossed for another near-clean run...

**Comment by mlilynolting** [comment]
jenkins retest

**Comment by mlilynolting** [comment]
jenkins retest

**Comment by mlilynolting** [comment]
Jigsawing the last 2 swat runs (JWF = Jupyter, Welder failed to start)

https://fc-jenkins.dsp-techops.broadinstitute.org/job/leonardo-fiab-test-runner/56443/consoleFull
  create runtime and attach a persistent disk - failed with JWF
  create runtime with Balanced disk - failed with JWF

https://fc-jenkins.dsp-techops.broadinstitute.org/job/leonardo-fiab-test-runner/56438/consoleFull
  NotebookAouSpec - failed with JWF
  NotebookGCEDataSyncingSpec - failed with JWF

Calling this passing.

**Pull request merged**